### PR TITLE
Removal of log4net from Binary and move to Microsoft.Extensions.Logging

### PR DESCRIPTION
**Pull request by Watson38205**

### CODE REVIEW

File: src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs [hunk 1]
```diff
@@ -14,11 +15,18 @@
...
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using opc.ua.pubsub.dotnet.binary.Storage;
 
 namespace opc.ua.pubsub.dotnet.binary.test
 {
     public class RoundTripWithSingleDataPoint
     {
+        private DecodeMessage m_DcodeMessage;
```
Comment (Modricagon) [inline src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs]: typo in name

**Review by Modricagon — CHANGES_REQUESTED** [review]

File: src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs (hunk 1 above)
Comment (Watson38205) [reply src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs]: Corrected type-o in name and removed un-used using statements.

File: src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs [hunk 2]
```diff
@@ -111,7 +106,7 @@ private static IEnumerable SingleItemTestCases
         public void Setup()
         {
             NullLogger<DecodeMessage> nullLogger = new NullLogger<DecodeMessage>();
-            m_DcodeMessage = new DecodeMessage( nullLogger );
+            m_DcodeMessage = new DecodeMessage( nullLogger, null, new EncodingOptions());
```
Comment (Modricagon) [inline src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs]: typo remains

**Review by Modricagon — CHANGES_REQUESTED** [review]

File: src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs (hunk 2 above)
Comment (Watson38205) [reply src/Tests/Binary.Test/RoundTripWithSingleDataPoint.cs]: Corrected type-o in file.

**Review by Modricagon — APPROVED** [review]

**Pull request merged**